servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestContents.java

```diff
@@ -376,4 +418,88 @@ public void multiget() throws BaseNessieClientServerException {
                     .isEqualTo(IcebergTable.builder().from(tableB).id(content.getId()).build()))
         .doesNotContainKey(ContentKey.of("noexist"));
   }
+
+  static Stream<Arguments> icebergMetadataScenarios() {
+    return Stream.of(loadIcebergMetadataScenario("table-three-snapshots", 3));
+  }
+
+  private static Arguments loadIcebergMetadataScenario(String name, int count) {
```

Comment:
This method is always called with the same args.

Reply:
Yea - only one "scenario" at the moment